# TODO(b/399031689):Clarify with NVIDIA if this is intentional, and whether it 
# will be changed in the future. Until then, this should be an internal patch.
diff --git a/third_party/nvidia/backend/compiler.py b/third_party/nvidia/backend/compiler.py
--- a/third_party/nvidia/backend/compiler.py
+++ b/third_party/nvidia/backend/compiler.py
@@ -51,7 +51,7 @@ def _path_to_binary(binary: str):
 
 @functools.lru_cache()
 def get_ptxas(arch: int):
-    name = "ptxas-blackwell" if arch >= 100 else "ptxas"
+    name = "ptxas"
     return _path_to_binary(name)
 
 
